src/tiledbsoma_ml/common.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2021-2024 The Chan Zuckerberg Initiative Foundation
```

Comment:
nit:  the file naming convention for "module-private" stuff is to prefix with underscore. I'm not sure if that is right or wrong, but we should be consistent.

Reply:
So should they all be exported from `__init__.py`?  (I realize this is a gray area as they are just types).

Totally up to you

Reply:
Interesting, I also don't know what makes most sense, but your suggestion sgtm so I went with it. These get more use in #27 as well.